### ISSUE TITLE
Can't seem to get the vim-airline working properly or the theme fully.

### ISSUE THREAD

**Issue by iamthesiz** [NextJump/jarvis#5]
Hey there!

I love this configuration.  I've been using it now for a couple of weeks.  Unfortunately I had to reformat my computer so I had to re-download everything.  After I did that I can't get things up and running for some reason.  Do you have any idea why this might not be working properly?

**Here are some screen-shots**
![screen shot 2014-04-08 at 1 45 07 am](https://cloud.githubusercontent.com/assets/5455859/2641457/22a24782-befa-11e3-82a0-61a97ce1832b.png)
![screen shot 2014-04-08 at 1 45 10 am](https://cloud.githubusercontent.com/assets/5455859/2641456/22a24174-befa-11e3-90bc-a2c8440a7e55.png)

**Comment by ctaylo21** [comment]
I'm glad you like it! I believe you need to download the patched font file specified in the dependencies section (that's why it looks like symbols are missing/weird).

**Comment by iamthesiz** [comment]
Okay, so I downloaded it.  I thought I already had it downloaded (which I did but it was just a directory sitting in a folder) but I went through the installation instructions on the documentation page.

After installing I tested it out and got this:
![screen shot 2014-04-08 at 10 31 02 am](https://cloud.githubusercontent.com/assets/5455859/2646824/9259339e-bf43-11e3-9e46-5fb996af21b3.png)

I then tried the following:
- removing the `jarvis/` directory  & reinstalling it
- after, quitting iTerm

Any ideas?

**Comment by iamthesiz** [comment]
Also, here's a snapshot of my home directory:
![screen shot 2014-04-08 at 10 34 10 am](https://cloud.githubusercontent.com/assets/5455859/2646886/1c8af250-bf44-11e3-814f-51f9ff462c5f.png)

**Comment by ctaylo21** [comment]
If you downloaded the font and installed it on your machine and it's still not working, the name of the font is probably wrong it the .vimrc. Make sure the font name is correct. Some of the patched font names have spaces which can be weird. If you are on a mac, in terminal you can click Terminal->Preferences and look for the font you installed. 

Also, I have been keeping Jarvis up on my personal account if you are interested. I don't know if it will ever be updated on this repo again, so if you want to get the most up to date stuff you can check it out. I always welcome contributors if you're interested. https://github.com/ctaylo21/jarvis

**Comment by iamthesiz** [comment]
Absolutely! I'm still learning vim but I always find `the good stuff` so I'll definitely contribute!

So I'm using iTerm and found the font within it.

I still can't seem to get it to work.  Here's what I added to the .vimrc

set guifont=Inconsolata-dz
execute pathogen#infect()
syntax enable
set background=dark
colorscheme solarized
let g:solarized_termcolors=256

**Comment by ctaylo21** [comment]
Are you using Vim or gVim? guifont is only for gVim, so it won't work if
you are just using Vim inside of iTerm.

**Comment by iamthesiz** [comment]
Getting somewhat closer it seems. Now we have a bar that goes across the bottom. What font do you use by the way? And where do you set the font?

.vimrc?
.vim/vimrc.local?
.vim/vimrc.custom?
.vim/vimrc?
elsewhere?

Here's a screenshot:
![screen shot 2014-04-08 at 1 31 15 pm](https://cloud.githubusercontent.com/assets/5455859/2648923/ce1cb798-bf5c-11e3-81b9-4a13c1f84ca7.png)

**Comment by ctaylo21** [comment]
I use Inconsolata and love it. If you are using straight up Vim, it should use whatever font your terminal/terminal emuator is set to. If you set your font to the airilne-patched font you download in iTerm, it should work in Vim. Setting your font in the vimrc will normally work for gVim but I'm not aware of a way to set a default in Vim using vim configurations. I just have my terminal settings to always use the patched font.

**Comment by iamthesiz** [comment]
I got it.  Here was the problem.  Basically I had the fonts installed but I wasn't using the correct one.  
eg. 
I had:      Inconsolata-dz
Needed: Inconsolata for Powerline

Just for future reference.  I read that the files basically have to have " for Powerline" after them otherwise they won't work.  Thanks for all your help!  And I'll shoot any great ideas/plugins/settings your way!

**Comment by iamthesiz** [comment]
One last question.  Why do the plugins not work unless I open vim like this:

```
$ cd path/to/directory
$ path/to/file❯vim
```

Why can't I open it like this?

```
$ vim path/to/directory
```

**Comment by ctaylo21** [comment]
Sorry, I'm not sure I understand the problem. You should be able to open vim like 

<code>vim path/to/directory</code>

Can you be more specific when you say the plugins don't work? Do you see any errors?

**Comment by iamthesiz** [comment]
Okay, don't worry about what I was saying before.  I got it figured out.  I was basically trying to automate the task but I just don't have a firm grasp on bash yet so I had to figure that out.  I had a couple questions though, not necessarily issues, just questions.

For `Sparkup` are you able to hit `tab` (like in Sublime Text) or some other shortcut and then it takes you to the next edit point? Just curious.

Is there a good FTP plugin that you recommend?
